### DISCUSSION TITLE
[Feature Request]: 微信通知影片入库消息仅通知单个成员

### DISCUSSION THREAD

**Discussion by rui-ao** [jxxghp/MoviePilot#3400] [讨论]
### 当前程序版本

1.5.4

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

为方便家人使用，使用微信通知，但是别人订阅的影片入库了会通知到所有人，能否仅通知订阅他的人，就像输入一个剧名之后搜索结果仅通知这个人一样，不知是否方便实现

**Comment by rdself** [comment]
同求，希望“开始下载”和“入库通知”能只通知点播的那个用户，不然家里人一起用的时候太干扰别人了。
感谢！
